README.md

```diff
@@ -25,10 +26,26 @@ Guide.
 
 ## Official releases
 
-Release versions of Spine Toolbox can be found 
+### With Python installed
+
+Spine Toolbox is available on [The Python Package Index (PyPI)](https://pypi.org/project/spinetoolbox/).
+With Python installed in your system, you can get the latest release using [pipx](https://pypa.github.io/pipx/).
+
+    pipx install spinetoolbox
+
+Start the application by running `spinetoolbox` which should now be available on the path.
+Update the application with 
+
+    pipx upgrade spinetoolbox
+
+
+### Without Python 
+
+Binary installer packages of Spine Toolbox can be found 
 [here](https://drive.google.com/drive/folders/1t-AIIwRMl3HiYgka4ex5bCccI2gpbspK)
```

Comment:
This link should be updated to `https://github.com/Spine-project/Spine-Toolbox/releases`.

Reply:
done